### PR TITLE
Upgrade to python 3

### PR DESCRIPTION
**Pull request by mojeto**
Python 3.6 chosen as a base line, because that's the highest version compatible with Wagtail 1.13.4.

Moving to Python 3.7 will be done together with upgrade of Wagtail, Django and all other dependencies in next PR.

### CODE REVIEW

**Comment by b-jam** [comment]
Once the settings parser thing is working, I'll PR add/infra so that theres a CI build to deploy to new infra

Then we can stop codeship on this project

**Comment by b-jam** [comment]
@haydnnewport or @mojeto could you please have a look?
Note the commits look like they were rebased because I had to use git filter-repo for some secrets I added into this branch.

File: madewithwagtail/settings/base.py [hunk 1]
```diff
@@ -1,220 +1,15 @@
-"""
-Django settings for  project.
+from .grains.database import *
+from .grains.django import *
+from .grains.il8n import *
+from .grains.email import *
+from .grains.huey import *
```
Comment (haydnnewport) [inline madewithwagtail/settings/base.py]: Let's remove this here (and delete the config all together) if it's not being used.

File: madewithwagtail/settings/__init__.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    ),  # Application environment i.e. development, production, etc.
+    StringVariable(
+        "APPLICATION_VERSION"
+    ),  # The Application version used across project
+    StringListVariable("ADDRESSES", default=""),
+    StringVariable("AWS_STORAGE_BUCKET_NAME", default=""),  # S3 Bucket Name
+    StringVariable("AWS_S3_CUSTOM_DOMAIN", default=""),  # S3 Domain
+    #StringVariable("CACHE_URL"),  # Cache url e.g. Redis cache URL
+    #StringVariable("TASK_QUEUE_URL"),  # Task queue url e.g. Redis URL
+    # StringVariable("ELASTICSEARCH_URL"), # Elastic search URL
```
Comment (haydnnewport) [inline madewithwagtail/settings/__init__.py]: Could we clean these up too, please?

**Pull request merged**